ui/raidboss/data/04-sb/dungeon/ghimlyt_dark.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheGhimlytDark,
+  timelineFile: 'ghimlyt_dark.txt',
+  timelineTriggers: [
+    {
+      id: 'Ghimlyt Dark Prometheus Laser',
+      regex: /Heat/,
+      beforeSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid laser from wall',
+        },
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'Ghimlyt Dark Jarring Blow',
+      netRegex: NetRegexes.startsUsing({ id: '376E', source: 'Mark III-B Magitek Colossus' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Ghimlyt Dark Wild Fire Beam',
+      netRegex: NetRegexes.headMarker({ id: '008B' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Ghimlyt Dark Ceruleum Vent',
+      netRegex: NetRegexes.startsUsing({ id: '3773', source: 'Mark III-B Magitek Colossus', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Ghimlyt Dark Magitek Ray',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Ghimlyt Dark Magitek Slash',
+      netRegex: NetRegexes.startsUsing({ id: '3774', source: 'Mark III-B Magitek Colossus', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid spinning laser',
```

Comment:
I think you could do "rotate left" or "rotate right" here, as I suspect most people will be facing the boss?

Reply:
Sure. I'll just need to double-check which marker is which. We must do research!